### ISSUE TITLE
Couldn't add filehandle from epoll set

### ISSUE THREAD

**Issue by eserte** [Grinnz/Mojo-Reactor-IOAsync#1]
In some situations the t/reactor_ioasync.t test fails:
```
Couldn't add filehandle from epoll set: Bad file descriptor at /home/cpansand/.cpan/build/2018022306/IO-Async-Loop-Epoll-0.18-2/blib/lib/IO/Async/Loop/Epoll.pm line 287.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 9 just after 80.
t/reactor_ioasync.t .... 
Dubious, test returned 9 (wstat 2304, 0x900)
All 80 subtests passed 
```

This seems to happen if
* IO::Async::Loop::Epoll is installed (probably linux-only, so cannot happen on other systems)
* Net::DNS::Native is installed
* perl >= 5.20 (at least I could not reproduce the failure with 5.10.1 - 5.18.4)

**Comment by Grinnz** [comment]
Seems to be the same issue as https://github.com/Grinnz/Mojo-Reactor-Epoll/issues/1. Tracked in https://rt.cpan.org/Ticket/Display.html?id=124932

**Comment by Grinnz** [comment]
This should be fixed by Linux::Epoll 0.013. In the next release the test will set the IO::Async::Loop backend to avoid this type of issue.
